File: backend/src/main/java/com/voiz/dto/ReviewResponseDto.java
package com.voiz.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import java.time.LocalDateTime;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class ReviewResponseDto {

    private int reviewIdx;
    private int menuIdx; 
    private int orderIdx;
    private String userId;
    private String guestName;
    private String comment;
    private int rating;
    private String nationality;
    private String language;
    private LocalDateTime createdAt;
    private String menuName; // optional: joined from VOYZ_MENUS
    
}

File: frontend/app/src/main/java/com/voyz/datas/model/dto/ReviewResponseDto.kt
package com.voyz.datas.model.dto

data class ReviewResponseDto(
    val reviewIdx: Int,
    val menuIdx: Int,
    val orderIdx: Int,
    val userId: String,
    val comment: String,
    val rating: Int,
    val nationality: String,
    val language: String?,
    val createdAt: String,
    val menuName: String? = null,
)


